### PR TITLE
Add procedure to remove AMQ Cert Manager

### PR DESCRIPTION
**Pull request by leifmadsen**
Resolves: rhbz#2006357

### CODE REVIEW

**Comment by JoanneOFlynn2018** [comment]
Will modularise this and create a new PR this week.

File: doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc [hunk 1]
```diff
@@ -87,3 +87,81 @@ catalogsource.operators.coreos.com "operatorhubio-operators" deleted
...
+----
+
+. Delete the ClusterServiceVersion from the `openshift-operators` namespace:
++
+[source,bash,options="nowrap"]
+----
+$ oc delete --namespace=openshift-operators csv amq7-cert-manager.v1.0.6
+
+clusterserviceversion.operators.coreos.com "amq7-cert-manager.v1.0.6" deleted
+----
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc:146]:
```suggestion
[source,bash,options="nowrap"]
----
$ oc delete --namespace=openshift-operators csv amq7-cert-manager.v1.0.6
----

Example output:

[source,bash]
----
clusterserviceversion.operators.coreos.com "amq7-cert-manager.v1.0.6" deleted
----
```

File: doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc [hunk 2]
```diff
@@ -87,3 +87,81 @@ catalogsource.operators.coreos.com "operatorhubio-operators" deleted
...
+
+. Optionally, delete the CustomResourceDefinitions related to the AMQ Cert Manager Operator:
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc:148]:
```suggestion
. Delete the CustomResourceDefinitions related to the AMQ Cert Manager Operator:
```

File: doc-Service-Telemetry-Framework/modules/proc_removing-the-amq-cert-manager-operator.adoc [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+[id="removing-the-amq-cert-manager-operator_{context}"]
+= Removing the AMQ Certificate Manager Operator
+
+[role="_abstract"]
+If you are not using the AMQ Certificate Manager for any other applications, delete the Subscription, ClusterServiceVersion, and CustomResourceDefinitions.
+
+.Procedure
+
+. Get the current ClusterServiceVersion installed:
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_removing-the-amq-cert-manager-operator.adoc:32]:
```suggestion
. Retrieve the version number of your installed ClusterServiceVersion:
```

**Review by jamsmithredhat — APPROVED** [review]
Looks good, Leif

**Pull request merged**